tests/riotboot_flashwrite/Makefile

```diff
@@ -1,6 +1,8 @@
-DEVELHELP ?= 0
+# If no BOARD is found in the environment, use this default:
```

Comment:
tests usually include ```../Makefile.tests_common```.
That sets BOARD and DEVELHELP to defaults, and also configures the application name to something that cannot clash. As is, this application would be called ```riotboot_flashwrite```, same as the module.

Reply:
It is still included I just moved it to the bottom. The first commit only does that, to group things better (although that is just my personal taste I guess), but I can always drop that commit.

Reply:
Ahok. Then please move it back to the top, for consistency. Just put the variables that override it above.

Reply:
Done.